examples/util/src/lib.rs

```diff
@@ -16,7 +17,13 @@ impl Drop for ChildGuard {
     }
 }
 
-pub fn run_prometheus(enable_exemplars: bool) -> ChildGuard {
+pub fn run_prometheus(enable_exemplars: bool) -> Option<ChildGuard> {
+    let system = System::new_all();
+
+    if system.processes_by_exact_name("prometheus").any(|_| true) {
```

Comment:
Actually, will this return something if it's running in Docker?

Reply:
Yeah I tested it on mac and it doesn't find the process. I believe the process name is `docker` :/

Reply:
As noted below - works for linux and native proccess - I think on other OS still have some value - can use already running instance.